lib/node_modules/@stdlib/math/base/complex/divide/lib/robust-internal.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+// MODULES //
+
+var internalCompreal = require( './internal-compreal' );
```

Comment:
When we `require` a file, we always include the filename extension to be explicit in what we are requiring. Here, it is not clear, from reading the code, whether we are requiring a file having a basename `internal-compreal` or an `index.js` file from a directory.